src/service/multisig.service.ts

```diff
@@ -0,0 +1,102 @@
+import {Address, AggregateTransaction} from 'nem2-sdk';
```

Comment:
Test multisig.service.ts

Reply:
OK! will need some time as I'm not familiar with chai

Reply:
added tests! please have a look!